src/components/LoanCollections/HomeExp/KivaLoanCardCategory.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner tw-text-center"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+		<template>
+			<div class="tw-hidden md:tw-grid md:tw-grid-cols-3 md:tw-gap-4">
+				<template v-for="(loanId, index) in augmentedLoanIds">
+					<new-home-page-loan-card
+						:item-index="index"
+						:key="`loan-${loanId}`"
+						:loan-id="loanId"
+					/>
+				</template>
+			</div>
+			<div class="tw-mt-4 tw-flex tw-flex-col md:tw-flex-row tw-justify-center tw-w-auto tw-mx-auto">
+				<kv-button
+					class="tw-mx-1 tw-mb-3 tw-whitespace-nowrap"
+					:variant="categoryButtonStyle"
+					:to="cleanUrl"
+				>
+					{{ viewAllLoansCategoryTitle }}
+				</kv-button>
+			</div>
+		</template>
+	</div>
+</template>
+
+<script>
+import { getCategoryName } from '@/util/categoryUtils';
+import NewHomePageLoanCard from '@/components/LoanCards/NewHomePageLoanCard';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+
+export default {
+	name: 'KivaLoanCardCategory',
+	components: {
+		KvLoadingSpinner,
+		NewHomePageLoanCard,
+		KvButton
+	},
+	props: {
+		loanIds: {
+			type: Array,
+			default: () => [],
+		},
+		selectedChannel: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	data() {
+		return {
+			name: '',
+			id: 0,
+			url: '',
+			browseButtonStyle: 'primary',
+			categoryButtonStyle: 'secondary',
+			lendByCategoryUrl: 'lend-by-category'
+		};
+	},
+	computed: {
+		isLoading() {
+			return this.augmentedLoanIds.length === 0;
+		},
+		augmentedLoanIds() {
+			const clonedLoanIds = [...this.loanIds];
+			// const promoCardId = 1;
+			// const loadMoreCardId = 2;
+			// TODO: splice if promoCard if active on row
+			// if (this.showPromoCard) {
+			// 	clonedLoanIds.splice(1, 0, promoCardId);
+			// }
+			// TODO: append loadMoreCard if active
+			// if (this.showLoadMoreCard) {
+			// 	clonedLoanIds.push(loadMoreCardId);
+			// 	return clonedLoanIds;
+			// }
```

Comment:
Will this commented out code be added? If not, this computed can be removed, and the `loanIds` used directly.

Reply:
> Will this commented out code be added? If not, this computed can be removed, and the `loanIds` used directly.

Agreed. I removed this in another carousel I built.